pom.xml

```diff
@@ -373,7 +374,7 @@
                 <plugin>
                     <groupId>com.mycila</groupId>
                     <artifactId>license-maven-plugin</artifactId>
-                    <version>4.1</version>
+                    <version>3.0</version>
```

Comment:
Downgrade?

Reply:
not a real downgrade, main needs to be merged in again :)